sample-auto-preview/build.gradle.kts

```diff
@@ -0,0 +1,50 @@
+plugins {
+  id("com.android.library")
+  id("org.jetbrains.kotlin.android")
+  id("io.github.takahirom.roborazzi")
+}
+
+roborazzi {
+  automaticPreviewScreenshots {
+    enabled = true
```

Comment:
Well, in my experience, Gradle plugins are likely to have an 'enable' property, which can be intuitive for users. It would be simpler without the 'enable' property, but this could lead to mistakes in the build logic or elsewhere. I don't have a strong opinion about this.